feat: add renderJavascript option (#134)
algolia/algoliasearch-netlify: plugin/README.md

  Putting "pathPrefix: /blog" will alias `/blog` to `/`
- `customDomain` - The custom domain that you use, if it's not possible to define it on your Netlify's settings.
  Putting "customDomain: example.com" will alias `example.com` to `<your-site-url>.netlify.app`

> - `renderJavascript` - _Default: `false`_ Use to enable the javascript rendering of your website, useful for SPA.

Reviewer: ```suggestion
- `renderJavascript` - _Default: `false`_ If true, we will use JavaScript to render your website. Useful for Single Page Applications.
```

>   See [documentation](https://www.algolia.com/doc/api-reference/crawler/configuration/render-java-script/) to understood the implication of this option.

Reviewer: ```suggestion
  See [documentation](https://www.algolia.com/doc/api-reference/crawler/configuration/render-java-script/) to understand the implication of this option.
```

>   See [documentation](https://www.algolia.com/doc/api-reference/crawler/configuration/render-java-script/) to understand the implication of this option.

Reviewer: ```suggestion
  See the [documentation](https://www.algolia.com/doc/api-reference/crawler/configuration/render-java-script/) to understand the implication of this option.
```
